code/game/gamemodes/changeling/powers/revive.dm

```diff
@@ -45,6 +45,36 @@
 		BITSET(H.hud_updateflag, STATUS_HUD)
 		BITSET(H.hud_updateflag, LIFE_HUD)
 
+		if(H.handcuffed)
+			var/obj/item/weapon/W = H.handcuffed
+			H.handcuffed = null
+			if(H.buckled && H.buckled.buckle_require_restraints)
+				H.buckled.unbuckle_mob()
+			H.update_inv_handcuffed()
+			if (H.client)
+				H.client.screen -= W
+			if(W)
+				W.loc = H.loc
```

Comment:
forcemove()

Reply:
You also shouldn't need to check if(W), because it's declared as a typecast copy of a variable whose existence is already enforced